src/app/[locale]/layout.tsx

```diff
@@ -10,12 +10,13 @@ import { Header } from '@src/components/templates/header';
 import initTranslations from '@src/i18n';
 import { locales } from '@src/i18n/config';
 
-export async function generateMetadata() {
-  const metatadata: Metadata = {
+export async function generateMetadata(): Promise<Metadata> {
```

Comment:
why is this a promise?

Reply:
I thought `generateMetadata` must be async. Apparently not: https://nextjs.org/docs/app/api-reference/functions/generate-metadata#regular-function